modules/ossm-migrating-workloads-using-the-istio-revision-label.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-revision-label_{context}"]
+= Migrating workloads using the Istio revision label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+Revision tags are not used in this example for simplicity. When migrating large meshes, you can use revision tags to avoid re-labeling all namespaces during future 3.y.z upgrades.
```

Comment:
```suggestion
Revision tags are not used in this example for simplicity. When migrating large meshes, you can use revision tags to avoid re-labeling all namespaces during future version 3 updates.
```
See https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#version-release-and-modification-identifiers
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update

Reply:
done